firebase-firestore/src/main/java/com/google/firebase/firestore/LoadBundleTaskProgress.java

```diff
@@ -27,39 +38,79 @@
   private int totalDocuments;
```

Comment:
Should `totalDocuments`, `bytesLoaded`, and `totalBytes` be declared as `final` as well?

Reply:
Yeah, good catch.